build

```diff
@@ -133,6 +139,14 @@ make_docs() {
     erl -noshell -eval \
         "ok = emqx_conf:dump_schema('$docdir', $SCHEMA_MODULE), \
          halt(0)."
+    local desc="$docdir/desc.en.hocon"
```

Comment:
```suggestion
    local desc="$docdir/desc.en.json"
```
?

Reply:
we want to keep using .hocon because
1. JSON is HOCON
2. For the file concatenated in `emqx_dashboard/priv`, it's hocon